pkg/signalmeow/groups.go

```diff
@@ -174,6 +174,9 @@ func (cli *Client) GetAuthorizationForToday(ctx context.Context, masterKey libsi
 		return nil, err
 	}
 	randomness, err := libsignalgo.GenerateRandomness()
+	if err != nil {
+		return nil, fmt.Errorf("unable to read randomness: %w", err)
```

Comment:
GenerateRandomness should probably just be changed to panic like all other random generating functions 🤔

Reply:
right, I guess if there's not enough randomness, things have gone horribly wrong